tests/testsuite.src/run_file.at

```diff
@@ -6573,7 +6573,8 @@ AT_CHECK([$COMPILE prog.cob], [0], [], [])
 AT_CHECK([COB_CURRENT_DATE="2015/02/06 16:40:52" $COBCRUN_DIRECT ./prog], [0],
 [Normal termination, ending status: 10
 ], [])
-AT_CHECK([gcdiff mini-report reference-report], [0], [], [])
+AT_CHECK([$SED -e 's/[[[:blank:]]][[[:blank:]]]*$//' mini-report > mini-report.sed], [0], [], [])
```

Comment:
What is this needed for? If it is indeed, then please add a comment why (following the idea that we try to always compare what we produce, not only "something similar"), otherwise you may be able to use trigraph or otherwise fix that.

Reply:
This comes from a difference between 3.x and trunk. In 3.x the produced output does not have any trailing spaces, while in trunk it does (padding to column 80 or so).

I could add this padding (and the `@&t@` quadrigraph) to each line of the expected output (reference-report), but I thought it was better to keep it identical to the one in 3.x. Hence I stripped those trailing spaces in the produced output instead.

Reply:
Let's put that to the TODO file and add a summary "this is a difference, for now we change it to 3.x format for comparison purpose" (maybe a _bit_ more in two lines) to the test source - and keep the SED.

Thanks!